pinot-common/src/main/java/org/apache/pinot/common/utils/config/QueryOptionsUtils.java

```diff
@@ -167,6 +167,10 @@ public static boolean isCollectGcStats(Map<String, String> queryOptions) {
     return Boolean.parseBoolean(queryOptions.get(QueryOptionKey.COLLECT_GC_STATS));
   }
 
+  public static String getQueryHash(Map<String, String> queryOptions) {
+    return queryOptions.getOrDefault(QueryOptionKey.QUERY_HASH, CommonConstants.Broker.DEFAULT_QUERY_HASH);
+  }
```

Comment:
Can we add a broker option to enable this feature by default?

Reply:
There is already a broker-level option to enable it by default. Check CommonConstants.java, we can enable it on broker by setting `pinot.broker.enable.query.fingerprinting = true`